src/i18n_excel_from_xml.py

```diff
@@ -8,95 +8,186 @@
+		row += 1
+	for node in nodesPlural:
+		pluralname = getNodeKey(node)
+		keyone = 'plural:' + pluralname + ':one'
+		keyother = 'plural:' + pluralname + ':other'
+		sheet.write(row, 0, keyone)
+		row += 1
+		sheet.write(row, 0, keyother)
 		row += 1
 
-def writeValues(sheet, masterNodeList, nodes, col):
+def writeMasterValues(sheet, nodes, nodesPlural):
+	row = 1
+	value = ''
+	for node in nodes:		
+		while node.hasChildNodes():
+			value = value + getNodeValue(node)
+		sheet.write(row, 1, value)
+		value = ''
+		row += 1
+	for node in nodesPlural:
+		while node.hasChildNodes():
+			value = value + getNodeValue(node)
+		sheet.write(row, 1, value)
+		value = ''
+		row += 1
+
+def writeValues(sheet, masterNodeList, masterNodeListPlural, nodes, nodesPlural, col):
 	row = 1
 	for node in masterNodeList:
 		key = getNodeKey(node)
 		value = getNodeValueByKey(nodes, key)
 		sheet.write(row, col, value)
 		row += 1
+	for node in masterNodeListPlural:
+		key = getPluralNodeKey(node)
+		value = getPluralNodeValueByKey(nodesPlural, key)
+		sheet.write(row, col, value)		
+		row += 1
 
-def writeNodesToColumn(sheet, masterNodeList, nodes, langCode, col):
+def writeNodesToColumn(sheet, masterNodeList, masterNodeListPlural, nodes, nodesPlural, langCode, col):
 	sheet.write(0, col, langCode)
-	writeValues(sheet, masterNodeList, nodes, col)
+	writeValues(sheet, masterNodeList, masterNodeListPlural, nodes, nodesPlural, col)
 
-def writeMasterNodeList(sheet, nodes):
-	writeMasterKeys(sheet, nodes)
-	writeMasterValues(sheet, nodes)
+def writeMasterNodeList(sheet, nodes, nodesPlural):
+	writeMasterKeys(sheet, nodes, nodesPlural)
+	writeMasterValues(sheet, nodes, nodesPlural)
 
-def getStringsXmlFiles(directory):
-	return glob.glob(os.path.join(options.directory, 'res/values-*/strings.xml'))
+def getStringsXmlFiles(directory, xmlFileName):
+	return glob.glob(os.path.join(options.directory, 'res/values-*/' + xmlFileName))
 
-def getMasterNodeList(directory):
+
+def getMasterNodeList(directory): 
 	xmlFile = glob.glob(os.path.join(options.directory, 'res/values/strings.xml'))[0]
+	xmlstringlist = getNodeList(xmlFile)
+	xmlstringlist.append(getNodeListPlural(xmlFile)) 
 	return getNodeList(xmlFile)
 
 def getLangCode(xmlFile):
 	(head, tail) = os.path.split(xmlFile)
 	(head, tail) = os.path.split(head)
 	return tail.replace('values-', '')
+	
+def getFileName(xmlFile):
+	(head, tail) = os.path.split(xmlFile)
+	return tail
+	
+def getNameForSheet(xmlFile):
+	(head, tail) = os.path.split(xmlFile)
+	return tail.replace('strings_', '')
+	
+def getAllStringFilesInDefaultFolder(directory):
+	xmlFiles = glob.glob(os.path.join(options.directory, 'res/values/strings*.xml'))	
+	return xmlFiles	
 
 if __name__ == '__main__':
 	(options, args) = getOptions()
-
+	#Create excel book
 	book = initWorkBook()
-	sheet = getSheet(book)
+	#Find all string*.xml in default values directory
+	xmlFiles = getAllStringFilesInDefaultFolder(options.directory);
```

Comment:
Redundant comment: method name (`getAllStringFilesInDefaultFolder`) is already telling what is being done here. It should be removed.